app/views/posts/show.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1><%= post.title %></h1>
+<% if post&.featured? %>
+  <p><%= t("posts.featured.label") %></p>
+<% end %>
+<p><%= post.published_at %></p>
+<p><%= post.tags.map(&:name).join(", ") %></p>
+<p><%= post.article.content %></p>
+
+<% post.categories.map do |category| %>
```

Comment:
What do you think about using `#each` here instead, given that the return value from the `#map` is not being used?